### PR TITLE
move palette to visualizer

### PR DESCRIPTION
**Pull request by hhaAndroid**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: mmdet/visualization/local_visualizer.py [hunk 1]
```diff
@@ -13,7 +13,92 @@
...
+        new_palette.append(tuple(color))
+    return new_palette
+
+
+def get_palette(palette: Union[List[tuple], str, tuple],
+                num_classes: int) -> List[Tuple[int]]:
+    """Get palette from various inputs.
+
+    Args:
+        palette (list[tuple] | str | tuple): palette inputs.
```
Comment (jbwang1997) [inline mmdet/visualization/local_visualizer.py:39]: List[tuple] or str or tuple

**Comment by jbwang1997** [comment]
Others look good to me.

**Comment by ZwwWayne** [comment]
Is there any intention of this PR? It would be better to update the PR message.

**Review by RangiLyu — APPROVED** [review]

**Comment by ZwwWayne** [comment]
This PR is closed as no plans to make the refactor

**Pull request closed without merging**